### ISSUE TITLE
Cannot stop MediaObject

### ISSUE THREAD

**Issue by konradowy** [actions-on-google/actions-on-google-nodejs#277]
In a nutshell: I'm not able to stop MediaObject programatically.

I have a MediaObject which is not the final response (I need to respond for the finish playing callback)
I have also SuggestionsChip "Finish" which suppose to finish currently played MediaObject and quit the conversation.
It works properly on simulator but not on a device (on Google Assistant app).

Scenario 1: MediaObject is about to start playing media (already presented with a picture)
1. Tap on "Finish" suggestion chip
2. The app quits the conversation but also starts playing audio (should not happen)

Scenario 2: MediaObject is already playing media 
1. Tap on "Finish" suggestion chip
2. Agent returns "Ok" (or something else randomly), pauses playing Media Object but stays in the conversation. (should not happen)
3. If i "Finish" it again, the app quits the conversation

**Comment by Thomas-A-Reinert** [comment]
See https://stackoverflow.com/questions/54221405/ending-a-conv-playing-a-media-response-doesnt-work-as-expected-it-pauses-the-a/

**Comment by igilham** [comment]
Yep, known issue. It appears that they never considered that you might want any control over media playback.

**Comment by ahmadabdullah247** [comment]
Same issue, I guess it would be great if they add play pause capability in mediaobject
